.github/workflows/deploy.yml

```
@@ -0,0 +1,35 @@
+      contents: read
+    environment: 'WordPress.org'
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v5
+
+      - name: Deploy to WordPress.org
+        uses: 10up/action-wordpress-plugin-deploy@stable
+        env:
+          SVN_USERNAME: ${{ secrets.SVN_USERNAME }}
+          SVN_PASSWORD: ${{ secrets.SVN_PASSWORD }}
+          SLUG: ${{ inputs.SLUG }}
```

actionlint (1.7.7)

property "slug" is not defined in object type {version: string}

(expression)